resources/docker/clion_remote.Dockerfile

```diff
@@ -12,10 +12,7 @@
 #   user@password
 
 
-#FROM centos:7
-# FROM hub.opensciencegrid.org/slate/slate-client-server:1.0.7
 ARG baseimage=hub.opensciencegrid.org/slate/slate-client-server:2.0.4
```

Comment:
I have just pushed `hub.opensciencegrid.org/slate/slate-client-server:2.1.0` with the cache env vars and `ccache`.

Reply:
> For our notes @sthapa also mentioned including the following in the GitHub workflows:
> 
> ```
> CCACHE_READONLY
> ```

We don't need that and probably shouldn't use that.  I mentioned that in the context of using a http cache but that's we aren't using right now.

Reply:
Updated the image version and removed ccache